### ISSUE TITLE
用find, get等method，return回来的数据没有className

### ISSUE THREAD

**Issue by EvanFung** [LunaGao/leancloud_flutter_plugin#19]
AVQuery queryProduct = AVQuery("Product");
      final response = await queryProduct.find();
      print(response);

I/flutter (28328): [{"className":null,"fields":"{\"createdAt\":\"2019-12-16T12:41:49.622Z\",\"description\":\"THIS IS FUCK YOU\",\"imageUrl\":\"https://live.staticflickr.com/4043/4438260868_cc79b3369d_z.jpg\",\"objectId\":\"5df77b8d43c257006cb64d7b\",\"price\":99.0,\"title\":\"FUCK\",\"updatedAt\":\"2019-12-16T12:41:49.622Z\"}"}]

这里的className显示为null

**Comment by zoezhou1999** [comment]
Hi EvanFung，do you solve this problem?

**Comment by zoezhou1999** [comment]
@EvanFung

**Comment by EvanFung** [comment]
> Hi EvanFung，do you solve this problem?

No, I don't use this plugin anymore, but you can think about the decorator pattern.
